### PR TITLE
Add JavaScript to the mobile menu. 

### PR DESCRIPTION
**Pull request by c3c1l1a**
This is the implementation [Figma template 5](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1411) Mobile menu
##### Features:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi <STUDENT_MENTION>, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have Javascript linters set up. Kindly make sure to set up linters according to HTML-CSS-javascript linters given in linters config frpom [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: styles.css [hunk 1]
```diff
@@ -30,8 +30,43 @@ html {
...
+
+.desktop-menu {
+  display: none;
+}
+
+.close-icon {
+  width: fit-content;
+  align-self: flex-end;
+  padding: 20px;
+}
```
Comment (thecodechaser) [inline styles.css:55]: - [ ] You did a good job on the project functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) design as much as possible. Please place some right margin to the close(X) button and make sure it matches with the Figma design.

### Your design

image

### Figma design 

image

File: styles.css [hunk 2]
```diff
@@ -30,8 +30,43 @@ html {
...
+  flex-direction: column;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  background-color: #fff;
+  background-image: url("images/image_geometry_menu_1.png"), url("images/image_geometry_menu_2.png");
+  background-position: top left, bottom right;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
```
Comment (thecodechaser) [inline styles.css:45]: - [ ] Please fix the following issues and make sure your design matches the Figma design.
   - Please reduce the size of the top left corner image.
   - Please reduce the top margin from the mobile menu options
   - Please use the correct font-weight for the mobile menu options.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @c3c1l1a, and @NayanKaran👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] Please install the correct linter configurations for this project HTML-CSS-JavaScript, You can download the linter config files from [Here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @c3c1l1a 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**